### PR TITLE
Job order style multi file upload for repositories and project proposals

### PR DESCRIPTION
**Pull request by NomadNaomie**

### CODE REVIEW

**Review by art29 — CHANGES_REQUESTED** [review]
Mostly good! Only issue is with the Drag & Drop, it goes under, which looks odd. If possible, try to add them over instead of under the Drag & Drop field.

**Comment by NomadNaomie** [comment]
I can't add it under the drag and drop unfortunately, we can take out the drag and drop but this is what it looks like
image

File: app/controllers/repositories_controller.rb [hunk 1]
```diff
@@ -98,7 +98,7 @@ def edit
 
   def create
     # @repository = @user.repositories.build(repository_params)
-
+    puts "repository_params: #{repository_params}"
```
Comment (art29) [inline app/controllers/repositories_controller.rb:101]: Probably remove that

File: app/javascript/packs/application.js [hunk 2]
```diff
@@ -162,6 +162,21 @@ window.examResponse = function (exam_id, answer_id) {
     }),
   });
 };
+window.dragndrop = function (event) {
+  event.preventDefault();
+  let images = [...document.getElementsByClassName("image-upload")];
+  console.log(event.dataTransfer.items);
```
Comment (art29) [inline app/javascript/packs/application.js:168]: Probably remove the console.log

**Review by art29 — APPROVED** [review]
You forgot a few debugging things, other than that, it's all good!

**Pull request merged**